dashing/distribution.yaml

```diff
@@ -3230,6 +3230,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_topic.git
       version: dashing-devel
     status: maintained
+  rt_usb_9axisimu_driver:
+    doc:
+      type: git
+      url: https://github.com/rt-net/rt_usb_9axisimu_driver.git
+      version: dashing-devel
+    release:
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/rt-net-gbp/rt_usb_9axisimu_driver-release.git
+      version: 2.0.0-2
+    source:
+      test_pull_requests: true
```

Comment:
@ShotaAk just to double check, have you configured your repository accordingly (see [here](http://wiki.ros.org/buildfarm/Pull%20request%20testing))?

Reply:
Thank you for your notice, I just updated the collaborator access right.

https://github.com/ros/rosdistro/pull/26460#discussion_r484596867